docs/blog/20231212-Daeraxa-DecemberUpdate.md

```diff
@@ -0,0 +1,78 @@
+
+### Package compatibility
+
+One downside to moving to a new version of Electron is that some community packages do use some of these same deprecated features of Electron that have given us so much trouble with some of the core packages. On the surface, this means that we will potentially introduce some breaking changes in Pulsar, causing some of these packages to no longer work - the [Hydrogen](https://web.pulsar-edit.dev/packages/hydrogen) package being a perfect example of this.
+
+However, part of the goal of the Pulsar project was to make sure that the huge number of community packages out there were retained and it is in no way our goal to deliberately break them. To that end, any package authors who find their packages affected by the updated Electron versions are free to reach out to us via any of our [social channels](https://pulsar-edit.dev/community.html). Likewise, if any users are affected by any of these changes and find themselves with an affected package that is no longer maintained by the original author, the same avenues of help and support will be available should you wish to fork and publish the package under your own account.
+
+We also have a plan we mentioned recently, the [Pulsar Cooperative](https://pulsar-edit.dev/blog/20231004-Daeraxa-OctoberUpdate.html#introducing-pulsar-cooperative) initiative, in order to have a community-maintained space for package development without the overhead of full-on ownership.
+
+## PPR website issues
+
+<img src="./assets/tools.png" height=200>
+
+Some people may have noticed we sporadically have some issues with the Pulsar Package Registry website displaying packages correctly. This is rare and tends to resolve itself fairly quickly once the faulty instance is dropped. [@confused-techie] has introduced some additional error logging to try and nail this issue down. The error page has also been updated to include some links for people to report the issues as and when they see them in order to help us capture and resolve this issue.
+
+## Community Spotlight
+
+<img src="./assets/spotlight.png" height=200>
+
+### Default file icons
+
+Community member [@tthaumaturgist] has produced a whole bunch of amazing new icons that can be used to show that a given file will be opened within Pulsar as well as identifying the type of file at a glance.
+
+<img src="./assets/file-icons.png" height=500>
+
+They have produced icons for both Windows and macOS, [@confused-techie] is currently working on implementing this within Pulsar for Windows. A huge thank you to them for this wonderful contribution!
+
+***
+
+And that brings this, the final community update of the year, to a close. As always, a big thank you to all of our community members and a special thanks to those who donate to the project and make this possible. We hope to see you again in 2024!
+
+
+[@maurício szabo]: https://github.com/mauricioszabo
```

Comment:
I'm more on gitlab these days, not sure if we want to update this :thinking:

Reply:
Entirely up to you, it is meant to link to your profile, if you would prefer that it is your GitLab account or even your website then I'm happy to update it, the same block of names is copied between every update.